web-ng/src/app/components/title-dialog/title-dialog.component.ts

```diff
@@ -37,14 +37,17 @@ export class TitleDialogComponent {
   }
 
   async onCreateProject() {
-    const projectId = await this.projectService.createProject(
-      this.projectTitleForm.get('title')?.value
-    );
+    let projectId = '';
     try {
-      this.navigationService.setProjectId(projectId);
+      projectId = await this.projectService.createProject(
+        this.projectTitleForm.get('title')?.value
+      );
     } catch (e) {
       console.warn('Project creation failed', e);
     }
-    this.matDialogRef.close();
+    if (projectId) {
```

Comment:
Can `createProject` return a falesy value? If not, can you remove the conditional and move this up inside the try block?

Reply:
Fixed now!